spring-cloud-dataflow-docs/src/main/asciidoc/howto.adoc

```diff
@@ -197,4 +201,124 @@ dataflow:>stream deploy foo --properties "deployer.transform.count=2,app.transfo
 ----
 The same syntax can be applied for `xpath()` SpEL-function when you deal with XML data.
 Any other custom SpEL-function can also be used, but for this purpose you should build a library with the `@Configuration` class containing an appropriate `SpelFunctionFactoryBean` `@Bean` definition.
-The target Spring Cloud Stream application starter should be re-packaged to supply such a custom extension via built-in Spring Boot `@ComponentScan` mechanism or auto-configuration hook.
\ No newline at end of file
+The target Spring Cloud Stream application starter should be re-packaged to supply such a custom extension via built-in Spring Boot `@ComponentScan` mechanism or auto-configuration hook.
+
+[[dataflow-jdbc-sink]]
+=== How to use JDBC-sink?
+The JDBC-sink can be used to insert message payload data into a relational database table. By default,
+it inserts the entire payload into a table named after `spring.application.name` value set at the
+application level, and if it is not set, by default the application expects the table with the name
+`messages`. To alter this behavior, the JDBC sink accepts link:https://github.com/spring-cloud-stream-app-starters/jdbc/blob/master/spring-cloud-starter-stream-sink-jdbc/README.adoc[several options] that you can pass using the --foo=bar notation in the stream, or change globally. You should also change the `jdbc.initialize` property to true to have this script execute when the sink starts up.
```

Comment:
what script do you refer to in "to have this script execute when the sink starts up", this last two sentences could need a bit of rewrite